### PR TITLE
add tests for large problem returning userlimit

### LINKED ISSUE #162: add unit test for first mip returning UserLimit
This corner case seemed to uncover a lot of small issues (https://github.com/mlubin/Pajarito.jl/pull/161 https://github.com/mlubin/Pajarito.jl/pull/160) not currently covered by our tests.

### PR DESCRIPTION
**Pull request by chriscoey**
5s and 15s on a slightly larger SDP problem

fixes #162

### CODE REVIEW

**Comment by chriscoey** [comment]
note Pajarito can't set time limit on GLPK solver

File: src/nonlinear_algorithm.jl [hunk 1]
```diff
@@ -132,7 +132,7 @@ end
 MathProgBase.hesslag_structure(d::InfeasibleNLPEvaluator) = MathProgBase.hesslag_structure(d.d)
 MathProgBase.initialize(d::InfeasibleNLPEvaluator, requested_features::Vector{Symbol}) =
 MathProgBase.initialize(d.d, requested_features)
-MathProgBase.features_available(d::InfeasibleNLPEvaluator) = [:Grad,:Jac,:Hess]
```
Comment (mlubin) [inline src/nonlinear_algorithm.jl]: Don't remove ``:Jac``.

Comment (chriscoey) [inline src/nonlinear_algorithm.jl]: what did you mean by
> You should also remove :JacVec from features_available.
on #18

Comment (mlubin) [inline src/nonlinear_algorithm.jl]: Well if it wasn't there in the first place there's nothing to do.

**Comment by chriscoey** [comment]
will fail until GLPK tagged 
https://github.com/JuliaOpt/GLPKMathProgInterface.jl/pull/39

**Comment by chriscoey** [comment]
should fix #162
